### PR TITLE
fix: incorrect rule in makefile

### PR DESCRIPTION
**Pull request by icyphox**
Removes missing make target `docs/package.json`.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -46,7 +46,7 @@ docs/cli:
 node_modules: package.json Makefile
 		npm ci
 
-docs/node_modules: docs/package.json
+docs/node_modules:
```
Comment (aeneasr) [inline Makefile:49]: This can be removed completely I suppose

**Comment by Benehiko** [comment]
should this not be merged to v0.8? Or will we just update it on v0.8 from master?

**Comment by aeneasr** [comment]
You're right, it should be against 0.8

**Comment by icyphox** [comment]
Okay well in that case I'll close and re-open against v8.

**Pull request closed without merging**